contracts/System.template

```diff
@@ -68,7 +69,7 @@ contract System {
     _;
   }
 
-  modifier onlyNotInit() {
+  modifier onlyNotInit() { //@openissue
```

Comment:
we can leave it as is but for future reference note that setting the flag in the modifier:
modifier onlyNotInit() {
   require( !alreadyInit );
   alreadyInit = true;
   _;
}

is considered a safer approach as it has a single point-of-failure and not many